udata/frontend/helpers.py

```diff
@@ -23,7 +25,11 @@
 
 @front.app_template_global(name='static')
 def static_global(filename):
-    return url_for('static', filename=filename)
+    if current_app.config['DEBUG']:
+        burst = time()
+    else:
+        burst = pkg_resources.get_distribution('udata').version
+    return url_for('static', filename=filename, _=burst)
```

Comment:
I took the same parameter as webpack, it makes short urls.
It will produce: ``http://xxx.com/static/filename?_=bust``.
It's not a flask parameter, flask adds the extra parameters (not registered in the url endpoint) as querystring